docs/docs/branching.md

```diff
@@ -50,10 +50,10 @@ The above diagram demonstrates retaining important historical snapshot with the
 via Spark SQL.
 
 1. Retain 1 snapshot per week for 1 month. This can be achieved by tagging the weekly snapshot and setting the tag retention to be a month.
-snapshots will be kept, and the branch reference itself will be retained for 1 week. 
+4 weekly snapshots will be kept, and the branch reference itself will be retained for 1 week. 
```

Comment:
Delete the first bullet point and append this after the last sentence before the list:
```suggestion
Assume snapshots are compressed to a single day before this command executes.

1. Create a tag on the snapshot occurring at the end of the first week, that will expire a month after it is created. You do this by setting the tag retention to be 30 days, or an average month. Run this command for each of the following weekend tag. 
```